src/main/scala/net/degoes/02-collections.scala

```diff
@@ -232,40 +238,46 @@ object Collections extends ZIOSpecDefault {
              * Using `List#foldLeft`, compute the sum of a list.
              */
             test("sum") {
-              def sum(list: List[Int]): Int = ???
+              def sum(list: List[Int]): Int = list.foldLeft(0)(_ + _)
 
               assertTrue(sum(List(1, 2, 3, 4, 5)) == 15)
-            } @@ ignore +
+            } +
               /**
                * EXERCISE
                *
                * Using `List#foldLeft`, compute the maximum element of a list.
                */
               test("max") {
-                def max(list: List[Int]): Int = ???
+                def max(list: List[Int]): Int = list.foldLeft(0)((x, y) => {
+                  case x > y => x
+                  case _ => y
+                })
 
                 assertTrue(max(List(1, 7, 3, 2, 4, 5)) == 7)
-              } @@ ignore +
+              } +
               /**
                * EXERCISE
                *
                * Using `List#foldLeft`, compute the minimum element of a list.
                */
               test("min") {
-                def min(list: List[Int]): Int = ???
+                def min(list: List[Int]): Int = list.foldLeft(Int.MaxValue)((x, y) => {
+                  case x > y  => y
+                  case _ => x
+                })
 
                 assertTrue(min(List(1, 7, 3, 2, 0, 4, 5)) == 0)
-              } @@ ignore +
+              } +
               /**
                * EXERCISE
                *
                * Using `List#foldLeft`, compute the reverse of a list.
                */
               test("reverse") {
-                def reverse[A](list: List[A]): List[A] = ???
+                def reverse[A](list: List[A]): List[A] = list.foldLeft(List.empty: List[A])((tList: List[A], num: A) => num :: tList)
```

Comment:
Instead of `List.empty: List[A]`, you can write `List.empty[A]`

Also you don't need to specify the types of `tList` and `num`